Sources/SwiftDocC/Semantics/Technology/TutorialTableOfContents.swift

```diff
@@ -11,22 +11,27 @@
 import Foundation
 import Markdown
 
-/// An overview of the educational materials under a specific technology or technology area.
-public final class Technology: Semantic, DirectiveConvertible, Abstracted, Redirected {
+/// A top-level page that organizes a collection of tutorials into a hierarchy of volumes and chapters.
+///
+/// ## See Also
+///
+/// - ``Volume``
+/// - ``Chapter``
+public final class TutorialTableOfContents: Semantic, DirectiveConvertible, Abstracted, Redirected {
```

Comment:
It'd be good to have a small description of the reasoning behind the name change in commit 38e87d057f7d28a4def043e6cbb3a1757333a8a1 (the one which renames the type), so that we can capture the context behind the change.

Reply:
Updated in https://github.com/swiftlang/swift-docc/pull/1039/commits/e5f5e252760ab072c775c5d4d90057853cd60f29